controller/pinctrl.c

```diff
@@ -2048,6 +2053,11 @@ pinctrl_handle_put_dhcp_opts(
                          IP_ARGS(*offer_ip));
             msg_type = DHCP_MSG_NAK;
         }
+        if (in_flow-> nw_src == htonl(0x00000000)) {
```

Comment:
Nit: s/-> /->/

Reply:
Also, it's probably nicer to use `htonl(INADDR_ANY)`.

Reply:
done